e2e/tests/functional/plugins/reloadAction/reloadAction.e2e.spec.js

```diff
@@ -122,4 +122,14 @@ test.describe('Reload action', () => {
     expect(fullReloadAlphaTelemetryValue).not.toEqual(afterReloadAlphaTelemetryValue);
     expect(fullReloadBetaTelemetryValue).not.toEqual(afterReloadBetaTelemetryValue);
   });
+
+  test('is disabled in Previews', async ({ page }) => {
```

Comment:
This is fine. We could have also updated the preview tests to be explicit about what options we expect to see in the three dot Action menu